install-dependencies.sh

```diff
@@ -31,11 +31,11 @@ if [[ $platform == "linux" ]]; then
   # These commands must be kept in sync with the installation instructions.
   sudo apt-get update
   sudo apt-get install -y git cmake build-essential autoconf curl libtool python-dev python-numpy python-pip libboost-all-dev unzip graphviz
-  sudo pip install ipython typing funcsigs subprocess32 protobuf==3.0.0a2 colorama graphviz
+  sudo pip install ipython typing funcsigs subprocess32 protobuf==3.0.0a2 colorama graphviz cloudpickle
```

Comment:
Will do, but any way to have that one "include" a list or something so this doesn't have to be done in the future?